kobuki_node/src/odometry.cpp

```diff
@@ -97,7 +97,7 @@ std::unique_ptr<geometry_msgs::msg::TransformStamped> Odometry::getTransform()
   odom_trans->transform.translation.z = 0.0;
   odom_trans->transform.rotation = odom_quat_;
 
-  return std::move(odom_trans);
```

Comment:
Looks good. For reference: https://stackoverflow.com/questions/4316727/returning-unique-ptr-from-functions.